src/main/java/org/broadinstitute/hellbender/utils/help/GATKHelpDocWorkUnitHandler.java

```diff
@@ -46,6 +65,7 @@ public GATKHelpDocWorkUnitHandler(final HelpDoclet doclet) {
      * @param currentWorkUnit the work unit for the feature being documented
      */
     @Override
+    @SuppressWarnings({"deprecation","removal"})
```

Comment:
Is this something that's going to need to be updated as soon as we go to 17?

Reply:
Yes, since I've now rebased this branch on the Java 17 code, I've removed these.